docs/source/yelpsoa_configs.rst

```diff
@@ -51,6 +51,13 @@ instance MAY have:
     namespace. For example ``canary`` instances can have ``nerve_ns: main`` to route
     their traffic to the same pool as the other ``main`` instances.
 
+  .. _networking_mode:
+
+  * ``networking_mode``: Specify which kind of 
```

Comment:
I would rather this be called "net" and use lower case words like "host" or "none" or "bridge", just like docker does.


Then inside our codebase, as necessary, we use the symbols marathon needs.